integration-tests/src/test/java/oracle/kubernetes/operator/ITOperator.java

```diff
@@ -277,6 +288,101 @@ public void testBCreateDomainWithDefaultValuesInSampleInputs() throws Exception
     logger.info("SUCCESS - testBCreateDomainWithDefaultValuesInSampleInputs");
   }
 
+  @Test
+  public void testDeleteOneDomain() throws Exception {
```

Comment:
I don't think this test belongs in the QUICKTEST subset.   QUICKTEST is a curated subset of tests run by Wercker and often for local runs too, and it only has a limited time to run.   

It looks like if a test doesn't belong to the QUICKTEST subset, it needs to 'skip' itself via: 

Assume.assumeFalse(
         System.getenv("QUICKTEST") != null && System.getenv("QUICKTEST").equalsIgnoreCase("true"));

Reply:
Will do. Didn't know that.